src/components/Button/DropdownButton.js

```diff
@@ -0,0 +1 @@
+export { DropdownButton as default } from 'react-bootstrap';
```

Comment:
does it make sense to add snapshots? this will ensure markup doesnt change on a newer upgrade of react-bootstrap?